python/cm/management/commands/loadcluster.py

```diff
@@ -84,6 +85,23 @@ def create_config(config):
         return None
 
 
+def create_file_from_config(obj, config):
+    if config is not None:
+        conf = config["current"]["config"]
+        proto = obj.prototype
+        for key, value in conf.items():
+            if isinstance(value, dict):
```

Comment:
We cannot trust this condition as we have types such as `structure` and `map`, I think you need to create files based on the specification, and take the values from the config